### PR TITLE
Alpine CVE fixes for 3.17.8, 3.18.7, 3.19.2 and 3.20.1

### PR DESCRIPTION
**Pull request by ncopa**

### CODE REVIEW

**Comment by whalelines** [comment]
Did the changes get pushed to the upstream? The actions are failing to find them.

File: library/alpine [hunk 1]
```diff
@@ -39,10 +39,10 @@ s390x-Directory: s390x/
 i386-Directory: x86/
 amd64-Directory: x86_64/
 
-Tags: 3.18.6, 3.18
+Tags: 3.18.7, 3.18
 Architectures: arm64v8, arm32v6, arm32v7, ppc64le, s390x, i386, amd64
 GitFetch: refs/heads/v3.18
-GitCommit: 08dbcabbd55e7430a326b582e544c6e1695126f3
+GitCommit: fb6cd4d1c0fc2b1b73c34a445d120c02c82752cd
```
Comment (yosifkit) [inline library/alpine:45]: The failing tests all mean that we are unable to proceed until https://github.com/alpinelinux/docker-alpine/tree/v3.18 contains this commit id (`fb6cd4d1c0fc2b1b73c34a445d120c02c82752cd`).

Comment (ncopa) [reply library/alpine:45]: whoops, I forgot push one branch. Sorry. It's done now.

**Comment by tsuna** [comment]
Small typo in one of the commits and title of the PR: it's Alpine 3.20.1 not 3.20.2 😉

**Comment by ncopa** [comment]
@yosifkit I think think this is ready to be merged now. Thanks!

**Review by LaurentGoderre — APPROVED** [review]

**Review by yosifkit — APPROVED** [review]

**Pull request merged**